lib/segment/src/types.rs

```diff
@@ -2420,6 +2420,73 @@ impl Filter {
     }
 }
 
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+pub enum SnapshotFormat {
+    /// Created by Qdrant `<0.11.0`.
+    ///
+    /// The collection snapshot contains nested tar archives for segments.
+    /// Segment tar archives contain a plain copy of the segment directory.
+    ///
+    /// ```plaintext
+    /// ./0/segments/
+    /// ├── 0b31e274-dc65-40e4-8493-67ebed4bcf10.tar
+    /// │   ├── segment.json
+    /// │   ├── CURRENT
+    /// │   ├── 000009.sst
+    /// │   ├── 000010.sst
+    /// │   └── …
+    /// ├── 1d6c96ec-7965-491a-9c45-362d55361e9b.tar
+    /// └── …
+    /// ```
+    Ancient,
```

Comment:
🦖🏺📜🗿

Do we still support unpacking these? If we do, let's keep it. Otherwise I'd suggest to remove this.

Reply:
Not sure if it works, but the code is still there.
https://github.com/qdrant/qdrant/blob/dee106d74625e891256ebe06c2c2e0515650e67e/lib/segment/src/segment/segment_ops.rs#L471-L474